be/src/runtime/runtime_filter_mgr.cpp

```diff
@@ -331,9 +395,9 @@ Status RuntimeFilterMergeControllerEntity::merge(const PMergeFilterRequest* requ
                                            std::to_string(request->filter_id()));
         }
     }
-    cnt_val = iter->second.first;
+    cnt_val = iter->second.cnt_val;
```

Comment:
recheck here the cnt_val really need be `std::shared_ptr<RuntimeFilterCntlVal>` or be `ref`